apps/console/src/features/applications/components/settings/sign-on-methods/script-based-flow/script-based-flow.tsx

```diff
@@ -225,13 +225,15 @@ export const ScriptBasedFlow: FunctionComponent<AdaptiveScriptsPropsInterface> =
      */
     const addSecretToScript = (secret:SecretModel): void => {
         const doc = editorInstance.getDoc();
-        //If a code segment is selected, the selected text is replaced.
+        const secretNameString = "\""+secret.secretName+"\"";
```

Comment:
Shall we use template strings.
```suggestion
        const secretNameString = `"${ secret.secretName }"`;
```